baselines/dom.generated.d.ts

```diff
@@ -3796,7 +3796,12 @@ declare var FocusNavigationEvent: {
 }
 
 interface FormData {
-    append(name: any, value: any, blobName?: string): void;
+    append(name: string, value: any, fileName?: string): void;
```

Comment:
Ah, looking in the full diff I understand better what you meant.

It seems you need to add FormDataEntryValue to the json file as a typedef (search "typedef" in it to find examples)